api/tests/views_tests.py

```diff
@@ -57,15 +57,6 @@ def prem_api_client(premium_user: User) -> APIClient:
     return client
 
 
-@pytest.mark.parametrize("format", ("yaml", "json"))
-@override_settings(API_DOCS_ENABLED=True)
-def test_swagger_format(client, format):
-    path = f"/api/v1/swagger.{format}/"
```

Comment:
@say-yawn @jwhitlock I don't remember why we have this view? is it for an automated tool?

Reply:
The test was added in https://github.com/mozilla/fx-private-relay/pull/1447, to fix https://github.com/mozilla/fx-private-relay/issues/1201.

You added the original URL path in https://github.com/mozilla/fx-private-relay/pull/1115, when Django REST Framework was added. So, only @groovecoder from September 2021 knows...